app/Http/Controllers/Api/SignupsController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Rogue\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+use Rogue\Services\SignupService;
+use Rogue\Repositories\PhotoRepository;
+use Rogue\Http\Transformers\SignupTransformer;
+
+// use Rogue\Http\Requests;
+
+class SignupsController extends ApiController
+{
+    /**
+     * @var \League\Fractal\TransformerAbstract;
+     */
+    protected $transformer;
+
+    /**
+     * The signup service instance.
+     *
+     * @var Rogue\Services\SignupService
+     */
+    protected $signups;
+
+    /**
+     * The photo repository instance.
+     *
+     * @var Rogue\Repositories\PhotoRepository
+     */
+    protected $photo;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract  $posts
+     * @return void
+     */
+    public function __construct(SignupService $signups, PhotoRepository $photo)
+    {
+        $this->signups = $signups;
+        $this->photo = $photo;
```

Comment:
Should `$this->photo` be changed to `$this->photos` so it's plural like `$this->signups`?